projects/igniteui-angular/src/lib/progressbar/templates/circular-bar.component.html

```diff
@@ -2,7 +2,12 @@
     viewBox="0 0 100 100" preserveAspectRatio="none" role="progressbar" aria-valuemin="0" [attr.aria-valuemax]="max" [attr.aria-valuenow]="value">
     <circle class="progress-circular__innercircle" cx="50" cy="50" r="46" />
     <circle #circle class="progress-circular__circle" cx="50" cy="50" r="46" />
-    <text #text class="progress-circular__text" [class.progress-circular__text--hidden]="!textVisibility" id="myTimer" text-anchor="middle" x="50" y="60">
-        {{textContent ? textContent: valueInPercent + '%'}}
+    <text #text [class.progress-circular__text--hidden]="!textVisibility" id="myTimer" text-anchor="middle" x="50" y="60">
+        <ng-container *ngTemplateOutlet="textTemplate ? textTemplate : defaultTextTemplate;">
```

Comment:
The passed user-defined template should get a context with the value of the progress bar.
Something along the lines of `{ $implicit: this.valueInPercent }` so the user can bind on it in the passed template.